post-docs/add-back-references.py

```diff
@@ -0,0 +1,108 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+from urllib.request import urlopen
+import semver
+
+docs_link = "https://raw.githubusercontent.com/apache/airflow/main/docs/apache-airflow/redirects.txt"
+docs_archive_path = "../docs-archive"
+apache_airflow_path = docs_archive_path + "/apache-airflow"
+stable_version_path = apache_airflow_path + "/stable.txt"
+new_docs_version = "2.5.1"
+
+# back reference html template - we will replace 'dummy-url' at runtime
+template = """
+<html><head><meta http-equiv="refresh" content="0; url=dummy_url"/></head></html>
+"""
```

Comment:
nit: instead of doing this replace you can just define a function with url as parameter:

```python
def get_redirect_content(url: str):
    return f"<html><head><meta http-equiv="refresh" content="0; url={url}"/></head></html>"
```

Reply:
Sure, we can do that. Made the changes